impala/hiveserver2.py

```diff
@@ -758,29 +759,46 @@ def threaded(func):
 
 def connect(host, port, timeout=None, use_ssl=False, ca_cert=None,
             user=None, password=None, kerberos_service_name='impala',
-            auth_mechanism=None, krb_host=None):
+            auth_mechanism=None, krb_host=None, use_http_transport=False,
```

Comment:
@lekv -- What is the difference between auth_mechanism=None and auth_mechanism=NOSASL? dbapi.py specifies these values:

`AUTH_MECHANISMS = ['NOSASL', 'PLAIN', 'GSSAPI', 'LDAP']`

NOSASL in the default in the connect() method from that file.